use_cases/init-process-for-fluent-bit/README.md

```diff
@@ -0,0 +1,80 @@
+# Init process for Fluent Bit on ECS, multi-config support
+
+
+
+### What is init process and multi-config support?
+
+Multi-config support means when you use a Fluent Bit image with `init` tag on ECS, you can use some config files conveniently to configure Fluent Bit. These config files can be your own files that you upload to S3, or you can use the [built-in config files](https://github.com/aws/aws-for-fluent-bit/tree/mainline/ecs) we provide directly. Init process achieves this feature. 
+The init process also injects ECS Task Metadata into the Fluent Bit container as environment variables:
+
+```
+AWS_REGION / ECS_LAUNCH_TYPE / ECS_CLUSTER / ECS_FAMILY
+ECS_TASK_ARN / ECS_TASK_ID / ECS_REVISION / ECS_TASK_DEFINITION
+```
+
+You can use them as env vars directly in the Fluent Bit config.
```

Comment:
Can you give a short example config snippet here showing how a user might use them

Reply:
Sure, will add it